pkg/kubelet/images/image_gc_manager.go

```diff
@@ -341,6 +344,10 @@ func (im *realImageGCManager) freeSpace(bytesToFree int64, freeTime time.Time) (
 			glog.V(5).Infof("Image ID %s is being used", image)
 			continue
 		}
+		if im.isImageInWhitelist(image) {
```

Comment:
I am not sure if we wanto to expect user to configure raw image ID in kubelet config like this ...

Reply:
the feature probably needs some discussion with @kubernetes/sig-node-feature-requests

Reply:
Maybe using image Name and Tag is a more reasonable option?
Image gc can use them to find image ID.

Reply:
The docker image names combination is super complex, this definitely deserves a discussion in sig-node.